Toolbar: add allowKeyboardNavigation and update KBN article (#8976)
DevExpress/devextreme-documentation: concepts/05 UI Components/Toolbar/05 Accessibility/15 Keyboard Navigation.md

    </tr>
    <tr>
        <td>Tab</td>

>         <td>Moves focus to the first Toolbar item or to the next focusable element after the Toolbar.</td>

Reviewer: The list is supposed to tell me what happens if the option is `true`. The `TAB` key would move focus to the next element regardless of that options. So I would only leave the part that relates to internal item navigation. 

```suggestion
        <td>If a Toolbar receives focus, it focuses its first available item.</td>
```

>         <td>Moves focus to the first Toolbar item or to the next focusable element after the Toolbar.</td>

Author: ```suggestion
        <td>If a Toolbar receives focus, it focuses its first available item.</td>
```

    </tr>
    <tr>
        <td>Shift + Tab</td>

>         <td>Moves focus to the last Toolbar item or to the previous focusable element before the Toolbar.</td>

Reviewer: ```suggestion
        <td>If a Toolbar receives focus, it focuses its last available item.</td>
```

    </tr>
    <tr>
        <td>Tab</td>
        <td>Moves focus into the Toolbar or to the next focusable element after the Toolbar.</td>
    </tr>
    <tr>
        <td>Shift + Tab</td>
        <td>Moves focus into the Toolbar from the end or to the previous focusable element before the Toolbar.</td>
    </tr>
    <tr>
        <td>Left Arrow / Right Arrow</td>
        <td>Moves focus to the previous or next available toolbar item.</td>
    </tr>
    <tr>
        <td>Up Arrow / Down Arrow</td>

>         <td>Moves focus between available toolbar items.</td>

Reviewer 2: That's not correct, on some components up and down arrows will open their drop downs, otherwise it won't move focus
